Clean up dependancy for flatbuffer_loader

qihqi opened the pull request:
Test Plan: waitforsandcastle

Differential Revision: D38445936



cc @EikanWang @jgong5 @wenzhe-nrv @sanchitintel

cccclai reviewed (approved):
Looks great! Thanks for the spending tireless effort on it

qihqi committed:
Clean up dependancy for flatbuffer_loader (#86041)

Summary: Pull Request resolved: https://github.com/pytorch/pytorch/pull/86041

Test Plan: waitforsandcastle

Reviewed By: cccclai

Differential Revision: D38445936

fbshipit-source-id: 14b283c16afe57f52e810e02cf2ccc70023f8916
